packages/manager/modules/enterprise-cloud-database/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "@ovh-ux/manager-enterprise-cloud-database",
+  "version": "0.0.1",
+  "description": "OVH Enterprise Cloud Databases product control panel",
```

```suggestion
  "description": "OVH Enterprise Cloud Database product control panel",
```
Is it `Database` or `Databases` ?  because it seems inconsistent

---

If it is `databases`the module should be renamed to `@ovh-ux/manager-enterprise-cloud-databases`and folder too.

---

'Databases' changed to 'Database'